lib/plugins/inspec-plugin-manager-cli/test/functional/inspec-plugin_test.rb

```diff
@@ -569,6 +569,8 @@ def test_refuse_install_when_plugin_on_exclusion_list
   end
 
   def test_error_install_with_debug_enabled
+    skip "this test requires bundler to pass" unless defined? ::Bundler
```

Comment:
Are there test conditions under which Bundler is not available? We require it to run the test suite...

Reply:
We don't require bundler to run the test suite. It runs fine without it with the exception of some of these test assumptions. 

Requiring bundler of our users and especially contributors makes total sense, but the developers on this project shouldn't be beholden to an unnecessary requirement like bundler, especially when it just slows us down.

```
4402 % time ntimes 10 bundle exec ~/.rbenv/versions/2.6.2/bin/ruby -Ilib:test -e 0

real	0m5.506s
user	0m4.504s
sys	0m0.883s
4403 % time ntimes 10 ~/.rbenv/versions/2.6.2/bin/ruby -Ilib:test -e 0

real	0m0.876s
user	0m0.694s
sys	0m0.146s
```

Some of these commits are effectively decoupling our tests from bundler.